src/sentry/incidents/endpoints/serializers/workflow_engine_incident.py

```diff
@@ -52,49 +97,136 @@ def get_incident_status(self, priority: int | None) -> int:
             raise ValueError("Priority is required to get an incident status")
         return self.priority_to_incident_status[priority]
 
-    def get_incident_activities(
+    def get_open_period_activities(
         self, open_period: GroupOpenPeriod
     ) -> list[IncidentActivitySerializerResponse]:
-        # TODO: Implement this
-        return [
-            {
+        # a metric issue will only have one openperiod because if it's reopened it'll make a new metric issue
+        # this won't actually work until we start writing to the table for metric issues (or are we planning a backfill? I can't remember)
+
+        open_period_activities = []
+        incident_identifier = "-1"  # temp until we add a column
+        # if we are here we have IncidentActivityType.CREATED and IncidentActivityType.DETECTED so fill those out
+        created = {
+            "id": "-1",  # TODO add lookup table to get this info
+            "incidentIdentifier": incident_identifier,
+            "type": IncidentActivityType.CREATED,
+            "value": None,
+            "previousValue": None,
+            "user": None,
```

Comment:
In the `IncidentActivity` table we only have data in the `user` and `comment` columns from 2020-03-04 - 2021-06-01, otherwise it's empty